specs/waku/waku-1.md

```diff
@@ -298,13 +303,13 @@ Each node SHOULD broadcast its rate limits to its peers using the rate limits pa
 
 Each node SHOULD respect rate limits advertised by its peers. The number of packets SHOULD be throttled in order not to exceed peer's rate limits. If the limit gets exceeded, the connection MAY be dropped by the peer.
 
-#### Message Confirmations update
+##### Message Confirmations Field
 
 Message confirmations tell a node that a message originating from it has been received by its peers, allowing a node to know whether a message has or has not been received.
 
-A node MAY send a message confirmation for any batch of messages received with a packet Messages Code.
+A node MAY send a message confirmation for any batch of messages received with a Messages packet (`0x01`).
 
-A message confirmation is sent using Batch Acknowledge packet or Message Response packet. The Batch Acknowledge packet is followed by a keccak256 hash of the envelopes batch data.
+A message confirmation is sent using Batch Ack packet (`0x0B`) or Message Response packet (`0x0C`). The message confirmation is specified in the ABNF specification below.
```

Comment:
The Batch Ack packet and message response packet are explained here. Is that ok or should we have that in a packet section?